Doc/library/sqlite3.rst

```diff
@@ -2358,6 +2345,88 @@ can be found in the `SQLite URI documentation`_.
 .. _SQLite URI documentation: https://www.sqlite.org/uri.html
 
 
+.. _sqlite3-howto-row-factory:
+
+How to create and use row factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` represents each fetched row as a :class:`tuple`.
+If a :class:`!tuple` does not suit your needs,
+use the :class:`sqlite3.Row` class or a custom :attr:`~Cursor.row_factory`.
```

Comment:
Initially, I was going to suggest stating more directly _how_ to use these (i.e. something like "set the `row_factory` attribute to `Row` or a custom row factory."), since reading it I was a little confused that I was being told to use something without being given the actionable information of _how_ to actually do that.

However, considering the next sentence where you describe where to set this, it seems to makes sense to hold off on stating this directly. Therefore, to avoid inducing any premature user confusion, I suggest rewording this to inform users that that using another row factory is an option, as opposed to directly telling them to do so.

```suggestion
you can use the :class:`sqlite3.Row` class
or a custom :attr:`~Cursor.row_factory`.
```